File: src/test/java/com/technews/scheduler/BlogSpringSchedulerTest.java
package com.technews.scheduler;

import org.junit.jupiter.api.Disabled;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.ActiveProfiles;

@Disabled
@ActiveProfiles("local")
@SpringBootTest
class BlogSpringSchedulerTest {

    @Autowired
    private BlogSpringScheduler blogSpringScheduler;

    @Test
    void run() {
        blogSpringScheduler.runScheduler();
    }
}

File: src/test/java/com/technews/scheduler/BlogSpringSchedulerTest.kt
package com.technews.scheduler

import com.technews.IntegrationTest
import io.kotest.core.spec.style.StringSpec
import org.junit.jupiter.api.Disabled
import org.springframework.beans.factory.annotation.Autowired

@Disabled
@IntegrationTest
class BlogSpringSchedulerTest(
    @Autowired private val blogSpringScheduler: BlogSpringScheduler,
) : StringSpec({
        "scrape a spring blog post" {
            blogSpringScheduler.runScheduler()
        }
    })
